CONTRIBUTING.md

```diff
@@ -216,48 +217,49 @@ Elements is used in the Stoplight Platform, as well as in open source projects.
 
 ## Releasing Elements
 
-1. Make sure you are in the root directory of this repo and on the `main` branch. Also pull all the latest changes!
-2. Create a new branch. The name doesn't really matter - something like `chore/release` will do.
-3. Run `yarn version`. You will be asked a few questions.
-    - We don't release `elements-demo`. In order to not release them, you have to choose "Custom Version" and enter the same, old verison by hand. Awkward, but it works.
+1. Make sure you are in the root directory of this repo and on the `main` branch.
+2. Pull all the latest changes.
+3. Create a new branch. The name doesn't really matter - something like `chore/release` will do.
+4. Run `yarn version`. You will be asked a few questions.
+    - We don't release `elements-demo`. In order to not release them, you have to choose "Custom Version" and enter the same, old version by hand. Awkward, but it works.
     - For other packages, you can just choose the proposed version (patch or minor update) from the console. Remember to read our versioning guidelines below!
-4. After the script gets finalized, a commit will be created for you.
-5. *IMPORTANT!!!* If `elements-core` version was updated in `elements` and/or `elements-dev-portal`, the script has changed `~` sign to `^` in front of `elements-core` version. Change it back to `~` in both places manually and make a new commit. The commit message doesn't really matter here, something like `fix: ^ to ~ for elements-core` is more than enough.
-6. Now make a push, create a PR and ask someone for a review. The purpose of the review is to make sure the release obeys our versioning guidelines.
+5. After the script gets finalized, a commit will be created for you.
+6. *IMPORTANT!!!* If `elements-core` version was updated in `elements` and/or `elements-dev-portal`, the script has changed `~` sign to `^` in front of `elements-core` version. Change it back to `~` in both places manually and make a new commit. The commit message doesn't really matter here, something like `fix: ^ to ~ for elements-core` is more than enough.
+7. Now make a push, create a PR and ask someone for a review. The purpose of the review is to make sure the release obeys our versioning guidelines.
 
 ## Versioning Guidelines
 
-If you did changes only in `elements` package, it's okay to release only `elements` package.
+If you did changes only in the `elements` package, it's okay to release only the `elements` package.
```

Comment:
Instead of "did changes" I would use "made changes" (here and in the next couple of lines)

Reply:
Oh great catch! I completely missed that 🤦 .